src/cli/simple/mod.rs

```diff
@@ -76,6 +100,9 @@ pub enum Command {
         function: String,
         #[arg(action, value_name = "input")]
         input: String,
+        /// Aurora EVM secret key
+        #[arg(long, value_name = "aurora-secret-key")]
```

Comment:
Yes. Because default naming is not good in some cases 😄 E.g.
Default:
```
 --engine <ENGINE>
          Aurora EVM account [default: aurora]
```
But with `value_name`, we can set <account> instead of <ENGINE> which makes more sense in this case.

Reply:
> value names should be in all capital pascal case

Yes. It makes sense.